src/collection-view.js

```diff
@@ -207,11 +207,13 @@ const CollectionView = Backbone.View.extend({
   // Render children views. Override this method to provide your own implementation of a
   // render function for the collection view.
   render() {
-    if (this._isDestroyed) { return this; }
-    this.triggerMethod('before:render', this);
-    this._renderChildren();
-    this._isRendered = true;
-    this.triggerMethod('render', this);
+    if (!this._isDestroyed) {
```

Comment:
The only way I can think we were missing code coverage on this is `this._isDestroyed` was always `false` in the tests. Would this change simply be giving us a false sense of testing security when we should be covering the `_isDestroyed` case too?